components/identity.oauth.uma.permission.endpoint/src/gen/java/org/wso2/carbon/identity/oauth/uma/endpoint/dto/ErrorResponseDTO.java

```diff
@@ -0,0 +1,61 @@
+package org.wso2.carbon.identity.oauth.uma.endpoint.dto;
+
+
+import io.swagger.annotations.*;
```

Comment:
Do not use * imports. Declate the individual imports

Reply:
this is auto generated